client/my-sites/checkout/composite-checkout/lib/get-plan-features.ts

```diff
@@ -0,0 +1,139 @@
+/**
+ * External dependencies
+ */
+import { useTranslate } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import { isEnabled } from 'calypso/config';
+import { isMonthly } from 'calypso/lib/plans/constants';
+import {
+	isWpComBusinessPlan,
+	isWpComEcommercePlan,
+	isWpComPersonalPlan,
+	isWpComPremiumPlan,
+} from 'calypso/lib/plans';
+import { WPCOMCartItem } from '../types/checkout-cart';
```

Comment:
```suggestion
import type { WPCOMCartItem } from '../types/checkout-cart';
```

It'd better to have `type` when it imports types only.
https://www.typescriptlang.org/docs/handbook/modules.html#importing-types